connection.go

```diff
@@ -85,5 +162,12 @@ func (c *Client) readWorker(errs chan error) { // readWorker works on a loop and
 		if msg != nil {
 			c.handleResponse(msg)
 		}
+
+		select {
+		case <-quit:
+			return
+		case <-time.After(10 * time.Millisecond):
```

Comment:
You'll execute a loop iteration every 10ms with this, right? That might be enough to peg one CPU core at 100%

Reply:
Not really, I added a quit channel to be able to exit gouroutine, if I don't add the time.After chan, the loop will stop on quit chan waiting a response, this function is reading a message from the WebSocket and will delay some time before getting to time.After. I will increase the value to 200 ms for be secure

Reply:
You don't need to add a delay in that case, having `default:` would be sufficient to exit the select clause.